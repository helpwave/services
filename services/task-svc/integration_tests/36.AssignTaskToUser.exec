#!/bin/bash

grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c", "user_id": "8aa8a5ee-2a35-4203-81bb-baf4ca17a9e9"}' \
	localhost:3001 \
	services.task_svc.v1.TaskService/AssignTaskToUser

grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c"}' \
	localhost:3001 \
	services.task_svc.v1.TaskService/GetTask

grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c"}' \
	localhost:3001 \
	services.task_svc.v1.TaskService/UnassignTaskFromUser

grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c"}' \
	localhost:3001 \
	services.task_svc.v1.TaskService/GetTask

grpcurl --plaintext -H "X-Organization: 3b25c6f5-4705-4074-9fc6-a50c28eba406" \
	-H "Authorization: Bearer eyJzdWIiOiIxODE1OTcxMy01ZDRlLTRhZDUtOTRhZC1mYmI2YmIxNDc5ODQiLCJlbWFpbCI6InRlc3RpbmUudGVzdEBoZWxwd2F2ZS5kZSIsIm5hbWUiOiJUZXN0aW5lIFRlc3QiLCJuaWNrbmFtZSI6InRlc3RpbmUudGVzdCIsIm9yZ2FuaXphdGlvbnMiOlsiM2IyNWM2ZjUtNDcwNS00MDc0LTlmYzYtYTUwYzI4ZWJhNDA2Il19" \
	-d '{"id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c", "user_id": "5db6cc86-bbf1-4fa6-bbe8-a8cdea869758"}' \
	localhost:3001 \
	services.task_svc.v1.TaskService/AssignTaskToUser
