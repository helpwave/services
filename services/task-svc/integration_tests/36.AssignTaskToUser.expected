{}
{
  "id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c",
  "name": "Changed Name",
  "description": "Clear description",
  "status": "TASK_STATUS_DONE",
  "assignedUserId": "8aa8a5ee-2a35-4203-81bb-baf4ca17a9e9",
  "patient": {
    "id": "8763d32d-73d1-4e53-b367-b970ee5f6c18",
    "name": "Grazia Rehn"
  },
  "public": true,
  "dueAt": "2023-11-04T06:58:00Z",
  "createdBy": "df363387-7774-4592-b01c-2b765abaa86a",
  "organizationId": "3b25c6f5-4705-4074-9fc6-a50c28eba406"
}
{}
{
  "id": "a2f04cb2-4587-48ad-89ee-a01256a80d7c",
  "name": "Changed Name",
  "description": "Clear description",
  "status": "TASK_STATUS_DONE",
  "patient": {
    "id": "8763d32d-73d1-4e53-b367-b970ee5f6c18",
    "name": "Grazia Rehn"
  },
  "public": true,
  "dueAt": "2023-11-04T06:58:00Z",
  "createdBy": "df363387-7774-4592-b01c-2b765abaa86a",
  "organizationId": "3b25c6f5-4705-4074-9fc6-a50c28eba406"
}
{}
